worker-service/src/main/java/com/ledger/owner/workerservice/web/WorkerController.java

```diff
@@ -0,0 +1,46 @@
+package com.ledger.owner.workerservice.web;
+
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.common.ledger.utils.LedgerResponse;
+import com.common.ledger.utils.ResponseUtil;
+import com.ledger.owner.workerservice.service.WorkerService;
+
+@RestController
+public class WorkerController {
+
+	WorkerService workerService;
+	
+	/**
+	 * 
+	 * @param wName param value expecting with worker name
+	 * @param wId param value expecting with worker id
+	 * @return
+	 */
+	@ResponseBody ResponseEntity<Map<String, Object>> getWorkerDetails(@RequestParam(name = "workerName", required=false) String wName,
+			@RequestParam(name = "workerId", required=false) String wId) {
+		
+		LedgerResponse ledgerResponse = null;
+		
+		if(StringUtils.isBlank(wName) && StringUtils.isBlank(wId)) {
```

Comment:
Can we try to remove if else if else blocks

Reply:
suggest which is better option to replace if else condition.. i too thought the same